src/app/beer_garden/garden.py

```diff
@@ -82,27 +82,21 @@ def local_garden(all_systems: bool = False) -> Garden:
     )
 
 
-def publish_garden(
-    event_name: str = Events.GARDEN_SYNC.name, status: str = "RUNNING"
-) -> None:
-    """Publish a Garden event
+@publish_event(Events.GARDEN_SYNC)
```

Comment:
That is sneaky. I also got mad about you not running the publish code until I saw this.